### DISCUSSION TITLE
Bootdisk Tdarr ise used more and more

### DISCUSSION THREAD

**Discussion by chicungunya** [tteck/Proxmox#2589] [Help Needed] [answered]
Hello,

![image](https://github.com/tteck/Proxmox/assets/25926287/ec11b621-e5fb-4ef1-8a0d-ecc8aa70229a)

I use a ramdisk for transcoding and it works. And the cache is removed after a transcode is succesfull.
But my bootdisk /dev/mapper/pve-vm--115--disk--0  increases gradually (little by little), why?

Thanks!

**Comment by tteck** [comment]
Have you tried the Proxmox VE LXC Filesystem Trim script?

**Reply by chicungunya** [reply]
I will. Should i do add a cron job for this every week?
& Should i use the Proxmox VE LXC Cleaner too?
Thanks!

**Reply by tteck** [reply]
First, see if _Proxmox VE LXC Filesystem Trim_ helps your issue.

**Reply by chicungunya** [reply, marked as answer]
Proxmox VE LXC Cleaner solved mine. How can i do a cronjob for this?
But why it increases? Because the logs I guess?
Thanks!
